hack/build.sh

```diff
@@ -52,17 +48,19 @@ build-docker() {
         --rm                                                              \
         -ti                                                               \
         -u "$(id -u)":"$(id -g)"                                          \
-        -v "$(pwd)"/.go:/go                                               \
+        -v "$(pwd)"/.go:/.go                                              \
```

Comment:
Why did you change `/go` into `/.go`? `/.go` is not in the `GOPATH` in the building container, it will download packages and modules every time when we use `make USE_DOCKER=1 build-client`.